tools/lint/src/main/java/com/grab/lint/LintAnalyzeCommand.kt

```diff
@@ -13,13 +21,21 @@ class LintAnalyzeCommand : LintBaseCommand() {
         workingDir: Path,
         projectXml: File,
         tmpBaseline: File,
-        lintBaselineHandler: LintBaselineHandler
     ) {
         val cliArgs = (defaultLintOptions + listOf(
             "--cache-dir", workingDir.resolve("cache").pathString,
             "--project", projectXml.toString(),
             "--analyze-only" // Only do analyze
         )).toTypedArray()
         LintCli().run(cliArgs)
+        sanitizePartialResults()
+    }
+
+    private fun sanitizePartialResults() = runBlocking {
+        partialResults.listFiles()?.asFlow()?.flatMapMerge { partialResult ->
```

Comment:
`flatMapMerge` uses `DEFAULT_CONCURRENCY` of 16 value which is too high for bazel actions which already run in parallel and might lead to thrashing. A good balance is `Runtime.getRuntime().availableProcessors / 2` similar to what's done [here](https://github.com/grab/grab-bazel-common/blob/4db9d71787fcfd546412382c4cdcc2ad25d7992e/tools/lint/src/main/java/com/grab/lint/LintBaseCommand.kt#L119).